src/oidcop/oidc/token.py

```diff
@@ -352,6 +363,274 @@ def post_parse_request(
 
         return request
 
+class TokenExchangeHelper(TokenEndpointHelper):
+    """Implements Token Exchange a.k.a. RFC8693"""
+
+    def __init__(self, endpoint, config=None):
+        TokenEndpointHelper.__init__(self, endpoint=endpoint, config=config)
+
+        if config is None:
+            self.config = {}
+        else:
+            self.config = config
+
+    def post_parse_request(self, request, client_id="", **kwargs):
+        request = TokenExchangeRequest(**request.to_dict())
+
+        _context = self.endpoint.server_get("endpoint_context")
+        if "token_exchange" in self.endpoint.kwargs:
+            self.config["token_exchange"] = _context.cdb[request["client_id"]].get(
+                "token_exchange", self.endpoint.kwargs["token_exchange"])
```

Comment:
Don't overwrite the object's config. 
```python
if "token_exchange" in _context.cdb[request["client_id"]]:
    self.config["token_exchange"] = _context.cdb[request["client_id"]]["token_exchange"]
else:
    config = self.endpoint.kwargs["token_exchange"]
```

Can `token_exchange` not exist in `self.endpoint.kwargs`? Would that allow us to enable token exchange only for specific clients while disabling it globally?